apps/trading/client-pages/markets/use-column-defs.tsx

```diff
@@ -126,7 +126,14 @@ export const priceValueFormatter = (
           data.decimalPlaces
         )} ${quoteAsset?.symbol}`;
   }
-  const quoteName = data && getQuoteName(data);
+  let quoteName: string | undefined = undefined;
+  if (data) {
+    try {
```

Comment:
Requiring a try catch here to me indicates something isn't quite right with the data object or if `data` can be undefined it would be better to just return early with an empty string